Update the-ember-times-issue-176.md (#936)
ember-learn/ember-blog: content/the-ember-times-issue-176.md

---

## [The ins and outs of organising EmberConf 📻](https://audio.ember.report/1680877/8115211-003-leah-silber-on-emberconf-2021)

> In the latest [Ember Report](https://twitter.com/EmberReport) [podcast]((https://audio.ember.report/1680877/8115211-003-leah-silber-on-emberconf-2021)) [Leah Silber (@wifelette)](https://twitter.com/wifelette) talks about organising EmberConf remote, the history of EmberConf and the topics like [**inclusion**](https://emberconf.com/about/inclusiveness-at-emberconf) that are so crucial to the EmberConf vibe.

Reviewer: I noticed that the link for `podcast` didn't show up. I made a few other suggestions that I think capture the meaning that you had. Can you have a look?

```suggestion
In the latest [Ember Report](https://twitter.com/EmberReport) [podcast](https://audio.ember.report/1680877/8115211-003-leah-silber-on-emberconf-2021), [Leah Silber (@wifelette)](https://twitter.com/wifelette) talks about organising a remote EmberConf, the history of EmberConf, and additional topics like [inclusiveness](https://emberconf.com/about/inclusiveness-at-emberconf) that are so crucial to the EmberConf vibe.
```

Reviewer 2: Looks good, thank you both! I'm hoping to submit a bit earlier today than normal, so am going to merge in these changes while have some cycles.

> This podcast gives a nice insight in the mind of a great conference organiser with some tips and pointers for others looking to organise conferences or meet-ups remote and in real live.

Reviewer: ```suggestion
This podcast gives a nice insight into the mind of a great conference organiser. There are also tips and pointers for people looking to organise conferences or meet-ups remotely and in real life.
```

> Besides information on how to organise a conference Leah also tells us about her favourite talks and the cool things that will happen this EmberConf including workshops and the [kids track](https://emberconf.com/kids).

Reviewer: ```suggestion
In addition to how to organise a conference, Leah tells us about her favourite talks and the cool things that will happen at EmberConf 2021, such as workshops and the [kids track](https://emberconf.com/kids).
```

After this paragraph, what do you think about adding a call to action? Something like:

```
Be sure to check out the podcast episode, [Leah Silber on EmberConf 2021](https://audio.ember.report/1680877/8115211-003-leah-silber-on-emberconf-2021)!
```
